CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 - Support python model through run command API, currently supported materializations are table and incremental. ([dbt-labs/dbt-spark#377](https://github.com/dbt-labs/dbt-spark/pull/377), [#126](https://github.com/databricks/dbt-databricks/pull/126))
+- Enable Pandas and Pandas-on-Spark DataFrames for dbt python models ([#181](https://github.com/databricks/dbt-databricks/pull/181))
```

Comment:
Could you also add the PR link in `dbt-spark`, like `[dbt-labs/dbt-spark#469](https://github.com/dbt-labs/dbt-spark/pull/469)`. See the above item.

Reply:
Good call. Just pushed that commit.